src/opnsense/mvc/app/library/OPNsense/System/Status/IDSOverrideStatus.php

```diff
@@ -47,17 +47,14 @@ public function __construct()
 
     public function collectStatus()
     {
-        $customFile = '/usr/local/etc/suricata/custom.yaml';
-        $expectedHash = '0cd7bf5ab0aa582303cfdebb3d4071366b232f300ef285101467533fd54c95d7';
+        $fileHash = @hash_file('sha256', '/usr/local/etc/suricata/custom.yaml');
+        $sampleHash = @hash_file('sha256', '/usr/local/opnsense/service/templates/OPNsense/IDS/custom.yaml.sample');
```

Comment:
almost, we check the template location exclusively, /usr/local/etc/suricata/custom.yaml will and should be overwritten anyway